opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -63,9 +63,17 @@ def __init__(
         super().__init__(instrumentation_info)
         self._instrumentation_info = instrumentation_info
         self._measurement_consumer = measurement_consumer
+        self._instrument_id_instrument = {}
```

Comment:
Is there some way to combine `self._instrument_id_instrument` here and `self._instrument_ids` in the `Meter` api? We have two collections that both hold "id" information.

Reply:
We would have to move the instrument_id checking into the SDK, which is a thing I think we want to have in the API. It is unfortunate, though that this information is repeated.

Reply:
Not a blocking question. As discussed, we can go ahead without this.